src/lib/Scenes/Artwork/Components/OtherWorks/index.tsx

```diff
@@ -1,37 +1,64 @@
+import { Box, Join, Separator, Spacer } from "@artsy/palette"
 import { OtherWorks_artwork } from "__generated__/OtherWorks_artwork.graphql"
+import GenericGrid from "lib/Components/ArtworkGrids/GenericGrid"
+import { filter } from "lodash"
 import React from "react"
 import { createFragmentContainer, graphql } from "react-relay"
-import { ArtworkContextArtistFragmentContainer as ArtworkContextArtist } from "./ArtworkContexts/ArtworkContextArtist"
+import { ContextGridCTA } from "./ContextGridCTA"
+import { Header } from "./Header"
 
-export const OtherWorksFragmentContainer = createFragmentContainer<{
-  artwork: OtherWorks_artwork
-}>(
+export const populatedGrids = grids => {
+  if (grids && grids.length > 0) {
+    return filter(grids, grid => {
+      const artworkNodes = grid.artworks && grid.artworks.edges && grid.artworks.edges.map(({ node }) => node)
```

Comment:
is there a reason to map over the edges here? Could maybe just do something like

    return grid.artworks && grid.artworks.edges && grid.artworks.edges.length > 0

Reply:
Ah! Good catch.